EventsByTag.md

```diff
@@ -40,39 +47,13 @@ persistenceId, tag combination. This sequence number can be generated as all the
 persistenceId are generated on the same node.
 
 If a gap is detected then the event is held back and the stream goes into a searching mode for the missing
-events. Depending on whether it is a live query and/or you have set an offset it might be possible
-to know for sure that an event is missing or it is just before the offset.
-
-Details for each of the four combinations of current/live and off set/no offset:
-* First event missing: The first found does not have sequence number 1
-* Event missing: Find some events in order then a sequence number jump
-* Omitted events: Can events be omitted or will the stream fail
-
-
-#### Current query / No offset
-
-* First event missing: Back track to `first-time-bucket`. N scans of each bucket.
-* Event missing: N scans from the previous event for this persistenceId/tag's offset to the current offset.
-* Omitted events: No, stream will fail if either of the above two scenarios don't find the missing event.
-
-#### Current query / Offset
+events. If the missing event is not found then then stream is failed.
 
-* First event missing: Back track to Offset. N scans of each bucket then assume it is the first event from this offset.
-* Event missing: N scans from the previous event for this persistenceId/tag's offset to the current offset.
-* Omitted events: Yes. If first event missing doesn't find an event that should be there.
-
-
-#### Live query / No Offset
-
-* First event missing: Back track to `first-time-bucket`. N scans of each bucket.
-* Event missing: N scans from the previous event for this persistenceId/tag's offset to the current offset.
-* Omitted events: No, stream will fail if either of the above two scenarios don't find the missing event.
-
-#### Live query / Offset
-
-* First event missing: Back track to Offset. N scans of each bucket then assume it is the first event from this offset.
-* Event missing: N scans from the previous event for this persistenceId/tag's offset to the current offset.
-* Omitted events: Yes. If first event missing doesn't find an event that should be there.
+When receiving the first event for a given persistenceId in an offset query it is not known 
+if that is actually the first event. If the offset query starts in a time bucket in the past then
+it is assumed to be the first. If the query is started with an offset in the current time bucket then
+the query starts with a scanning mode to look for the lowest sequence number per persistenceId
```

Comment:
lowest sequence number in that time bucket? or is it looking in previous bucket also?

Reply:
It only looks in the current bucket as anything in the previous bucket would have a timeuuid before the offset. 

When we look for other missing events we do look in the previous time bucket.